uxarray/grid/grid.py

```diff
@@ -119,6 +119,9 @@ def __init__(self,
         self._gdf = None
         self._poly_collection = None
         self._line_collection = None
+        self._centroid_points_df_proj = [None, None]
+        self._corner_points_df_proj = [None, None]
```

Comment:
centroid is face centered data and corner is node data, is that correct?

Reply:
Yes